src/anm/graphics/sheet.js

```diff
@@ -171,6 +171,22 @@ Sheet.prototype.bounds = function() {
     return new Bounds(0, 0, r[2], r[3]);
 };
 /**
+ * @method inside
+ *
+ * Checks if point is inside the image. _Does no test for bounds_, the point is
+ * assumed to be already inside of the bounds, so check `image.bounds().inside(pt)`
+ * before calling this method manually.
+ *
+ * @param {Object} pt point to check
+ * @param {Number} pt.x
+ * @param {Number} pt.y
+ * @return {Boolean} is point inside
+ */
+ Sheet.prototype.inside = function(pt) {
+    return true; // if point is inside of the bounds, point is considered to be
+                 // inside the image shape
+};
+/**
```

Comment:
you shall have to explain this to me in more detail, i'm afraid i don't get what this is for